package.json

```diff
@@ -123,7 +123,7 @@
     "zone.js": "^0.8.12"
   },
   "dependencies": {
-    "@angular/cdk": "^2.0.0-beta.8",
+    "@angular/cdk": "^2.0.0-beta.12",
```

Comment:
We should lock this to version `2.0.0-beta.12` so it won't break if a new release is published with breaking changes. This can probably also be moved to the `devDependencies`.